pkgs/top-level/python-aliases.nix

```diff
@@ -220,6 +220,7 @@ mapAliases ({
   pdfposter = throw "pdfposter was promoted to a top-level attribute"; # Added 2023-06-29
   pdfminer = pdfminer-six; # added 2022-05-25
   pep257 = pydocstyle; # added 2022-04-12
+  pipx = throw "pipx was promoted to a top-level attribute"; # added 2023-08-01
```

Comment:
```suggestion
  inherit (super.pkgs) pipx;
```

Reply:
That may be misleading because the python version won't be the same